### PR TITLE
Add option for performing HotelManager TXs asynchronously

### PR DESCRIPTION
**Pull request by adklempner**

### CODE REVIEW

File: libs/HotelManager.js [hunk 1]
```diff
@@ -188,7 +193,11 @@ class HotelManager {
     const estimate = await this.web3.eth.estimateGas(options);
     options.gas = await utils.addGasMargin(estimate, this.context);
 
-    return this.web3.eth.sendTransaction(options);
+    if(sync) {
```
Comment (AugustoL) [inline libs/HotelManager.js]: here it shouldnt be `return utils.execute(data, index, this.context, sync);` ?

Comment (adklempner) [reply libs/HotelManager.js]: It used `sendTransaction` before, but I'll see if using `execute` makes sense.

**Comment by adklempner** [comment]
Hmm I hadn't thought of listening for those callbacks within the library. I'll try that out.

File: libs/HotelManager.js (hunk 1 above)
Comment (AugustoL) [reply libs/HotelManager.js]: :+1:

Comment (adklempner) [reply libs/HotelManager.js]: `utils.execute` is for calling `callHotel`, so `sendTransaction` makes sense here and in `HotelManager.removeHotel`

Comment (AugustoL) [reply libs/HotelManager.js]: :+1:

**Pull request merged**